crates/bevy_core_pipeline/src/motion_blur/motion_blur.wgsl

```diff
@@ -0,0 +1,83 @@
+#import bevy_pbr::prepass_utils
+#import bevy_pbr::utils
+#import bevy_core_pipeline::fullscreen_vertex_shader FullscreenVertexOutput
+
+#ifdef MULTISAMPLED
```

Comment:
Multisampling specialization certainly makes things a bit more complex. I think this is a necessary evil though.